catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -328,6 +330,7 @@
 
     <feature name="catalog-app" install="auto" version="${project.version}"
              description="The DDF Catalog provides a framework for storing, searching, processing, and transforming information.\nClients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.\nAt the core of the Catalog functionality is the Catalog Framework, which routes all requests and responses through the system, invoking additional processing per the system configuration.::DDF Catalog">
+        <feature prerequisite="true">security-services-app</feature>
```

Comment:
I'm curious if there are more specific features we can should prereq on vs the whole app to speed up boot time (but that should really only matter in platform/security/admin).  Not sure it's really going to make a difference though since the ApplicationService only installs one app at and waits for it to finish installing.